src/test/scala-2.11/markets/participants/TestLiquiditySupplier.scala

```diff
@@ -19,11 +19,11 @@ import akka.actor.{Props, ActorRef}
 import akka.agent.Agent
 
 import markets.orders.Order
-import markets.participants.strategies.TestLimitOrderTradingStrategy
+import markets.participants.strategies.FixedLimitOrderTradingStrategy
 import markets.tickers.Tick
 import markets.tradables.Tradable
 
-import scala.collection.{immutable, mutable}
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2357985507/issues/source?bid=2921361&fileBranchId=3219434#l26)